charts/kyverno/templates/_helpers.tpl

```diff
@@ -107,11 +107,11 @@ maxUnavailable: {{ .Values.podDisruptionBudget.maxUnavailable }}
 
 {{- define "kyverno.replicaCount" -}}
 {{- if not (empty .Values.replicaCount) }}
-replicas: {{ .Values.replicaCount }}
+{{- printf "%v" .Values.replicaCount -}}
```

Comment:
:+1: for this, this template should return only the value.

Reply:
But why are you using `printf` here ?

Reply:
I'll try rendering with your suggestion

Reply:
This works, thanks